README.md

```diff
@@ -44,7 +44,56 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
 
 
 ### Marketing
-
+The objective of the marketing team is to create awareness of GeekcampSG
+
+#### Expectations
+Marketing team members need to manage and utilise social media and mailing lists to reach out to target segments. It is ok to be unfamiliar with the tools, but it is highly recommended that interested candidates should at least have some experience with using social media.
+
+#### People
+Assuming all templates are in place:
+- Pre-conference: Min 1 pax. Ideally at least 2 pax
+- Conference: Min 2 pax. Ideally at least 3 pax
+
+Note: during D day. There is a lot of work to be done. The lesser people there are for marketing, the less capacity there is to execute the full marketing plan
+
+#### Tasks
+
+##### Access
+- Give trusted members of the team access to the social media accounts
+
+##### Notifications
+- Setup email forwarding for new marketing team members
+- Setup tweetdeck to monitor hashtags and mentions
+
+##### Queries
+- Ensure that queries sent via email and on social media are answered promptly
+
+##### Prepare messaging templates for event phases
+- Recruitment
+- Team meeting pictures
+- Sponsorship deck
+- CFP + conference date
+- CFP closing soon / CFP extension
+- Confirmed schedule
+- Registration
+- Sponsor shoutouts
+- Reminder of event at D01
+- D-day postings for each talk: start and end
+- Sponsor booths
+- Food
+- After party
+- Post event Survey
+- Video uploads for each talk
+- Video playlist link
+
```

Comment:
- Post-event promotion (including video links above)

Reply:
@alanswanht I dont quite understand what you are referring to. Can you please give an example?

Reply:
Eg: The tweets and FB posts we have just scheduled for the videos and playlist.